### PR TITLE
[CGPROD-2321] Add particles on hovering select cells

### PR DESCRIPTION
**Pull request by AdamXA**

### CODE REVIEW

File: src/components/select/select-particles.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/**
+ * @copyright BBC 2020
+ * @author BBC Children's D+E
+ * @license Apache-2.0
+ */
+import { gmi } from "../../core/gmi/gmi.js";
+
+const onPointerOut = (emitter, button) => button.on(Phaser.Input.Events.POINTER_OUT, () => emitter.stop());
```
Comment (FostUK) [inline src/components/select/select-particles.js:8]: button.on(Phaser.Input.Events.POINTER_OUT, () => emitter.stop());
does 
button.on(Phaser.Input.Events.POINTER_OUT, emitter.stop);

work? might get binding errors...

File: src/components/select/select-particles.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+const createEmitter = (scene, button, config) => {
+    const emitter = scene.add
+        .particles(config.assetKey)
+        .createEmitter(scene.cache.json.get(config.emitterConfigKey))
+        .setPosition(button.x, button.y);
+    onPointerOut(emitter, button);
+};
+
+export const addHoverParticlesToCells = (scene, cells, config, layoutRoot) => {
+    if (!config) {
```
Comment (FostUK) [inline src/components/select/select-particles.js:19]: Can do if (!config) return; 

if you want

File: src/components/select/select-particles.js (hunk 1 above)
Comment (AdamXA) [reply src/components/select/select-particles.js:8]: na I get binding errors

File: src/components/select/select-particles.js (hunk 2 above)
Comment (AdamXA) [reply src/components/select/select-particles.js]: nice! 3 lines always felt too much for this! pretty sure I've done it elsewhere too

**Pull request merged**